### PR TITLE
Random fixes

### PR DESCRIPTION
**Pull request by AndreasElia**

### CODE REVIEW

File: database/migrations/2017_05_29_133643_create_discussions_table.php [hunk 1]
```diff
@@ -17,12 +17,13 @@ public function up()
             $table->increments('id');
             $table->string('title');
             $table->string('slug');
+            $table->integer('group_id')->unsigned();
```
Comment (NowNewStart) [inline database/migrations/2017_05_29_133643_create_discussions_table.php]: This makes tests fail. Are you fixing that yourself or is @deleugpn doing that after it got merged?

Comment (deleugpn) [inline database/migrations/2017_05_29_133643_create_discussions_table.php]: I may work on it.

Comment (NowNewStart) [inline database/migrations/2017_05_29_133643_create_discussions_table.php]: This seems to also fail the other tests. I believe an `->onDelete('cascade')` would fix this. Would also only make sense.

File: resources/views/discussions/create.blade.php [hunk 2]
```diff
@@ -8,27 +8,49 @@
...
-                <div class="form-group">
+                <div class="form-group{{ $errors->has('group_id') ? ' has-error' : '' }}">
                     <label for="group_id">Group</label>
                     <select name="group_id" id="group_id" class="form-control">
                         <option selected disabled>-</option>
 
                         @if (count($groups))
                             @foreach ($groups as $group)
-                                <option value="{{ $group->id }}">{{ $group->name }}</option>
+                                @if (old('group_id') == $group->id)
```
Comment (NowNewStart) [inline resources/views/discussions/create.blade.php]: Not sure if I'm alone here, but this isn't used just once. We should maybe add a Blade directive to make the blade templates cleaner.

Comment (Dylan-DPC) [inline resources/views/discussions/create.blade.php]: @NowNewStart feel free to create the directive

File: database/factories/ModelFactory.php [hunk 3]
```diff
@@ -8,8 +8,9 @@
 /* @var \Illuminate\Database\Eloquent\Factory $factory */
 $factory->define(Discussion::class, function (Faker\Generator $faker) {
     return [
-        'title' => $faker->sentence(6),
-        'slug'  => $faker->word,
+        'group_id' => factory(Group::class),
```
Comment (Dylan-DPC) [inline database/factories/ModelFactory.php]: same issue as the one Julien pointed out below

File: resources/views/discussions/edit.blade.php [hunk 4]
```diff
@@ -8,23 +8,45 @@
             <form action="{{ route('forum.discussions.update', $discussion->id) }}" method="PUT">
                 {{ csrf_field() }}
 
-                <div class="form-group">
+                <div class="form-group"{{ $errors->has('group_id') ? ' has-error' : '' }}>
                     <label for="group">Group</label>
                     <select name="group" id="group" class="form-control">
                         @foreach ($groups as $group)
-                            <option value="{{ $group->slug }}"{{ ($discussion->group->slug != $group->slug) ?: ' selected' }}>{{ $group->name }}</option>
+                            @if ($discussion->group_id == $group->id)
```
Comment (Dylan-DPC) [inline resources/views/discussions/edit.blade.php]: what's wrong with the ternary?

File: resources/views/discussions/create.blade.php (hunk 2 above)
Comment (NowNewStart) [inline resources/views/discussions/create.blade.php]: Will do after this got merged

Comment (Dylan-DPC) [inline resources/views/discussions/create.blade.php]: approve this change

Comment (NowNewStart) [inline resources/views/discussions/create.blade.php]: I just commented on it, no need to approve. You're the only one that needs to approve.

Comment (Dylan-DPC) [inline resources/views/discussions/create.blade.php]: cool then ;)

**Pull request merged**